sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Processor/EventProcessorHost.cs

```diff
@@ -4,42 +4,35 @@
 using System;
 using System.Collections.Concurrent;
 using System.Collections.Generic;
-using System.Globalization;
-using System.IO;
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
-using System.Text.Json;
-using Azure;
 using Azure.Messaging.EventHubs;
-using Azure.Messaging.EventHubs.Consumer;
 using Azure.Messaging.EventHubs.Primitives;
 using Azure.Messaging.EventHubs.Processor;
 using Azure.Storage.Blobs;
-using Azure.Storage.Blobs.Models;
-using System.Text.Json.Serialization;
 
 namespace Microsoft.Azure.WebJobs.EventHubs.Processor
 {
     internal class EventProcessorHost
     {
-        private string EventHubPath { get; }
-        private string ConsumerGroupName { get; }
-        private string EventHubConnectionString { get; }
-        private string StorageConnectionString { get; }
+        public string EventHubName { get; }
+        public string EventHubPath { get; }
+        public string ConsumerGroupName { get; }
+        public string EventHubConnectionString { get; }
+        public string StorageConnectionString { get; }
         private string LeaseContainerName { get; }
-        private string LegacyCheckpointStorageBlobPrefix { get; }
         private Processor CurrentProcessor { get; set; }
         private Action<ExceptionReceivedEventArgs> ExceptionHandler { get; }
 
-        public EventProcessorHost(string eventHubPath, string consumerGroupName, string eventHubConnectionString, string storageConnectionString, string leaseContainerName, string legacyCheckpointStorageBlobPrefix, Action<ExceptionReceivedEventArgs> exceptionHandler)
+        public EventProcessorHost(string eventHubName, string eventHubPath, string consumerGroupName, string eventHubConnectionString, string storageConnectionString, string leaseContainerName, Action<ExceptionReceivedEventArgs> exceptionHandler)
         {
+            EventHubName = eventHubName;
             EventHubPath = eventHubPath;
             ConsumerGroupName = consumerGroupName;
             EventHubConnectionString = eventHubConnectionString;
             StorageConnectionString = storageConnectionString;
             LeaseContainerName = leaseContainerName;
```

Comment:
Is `leaseContainer` this something that you're planning on removing?   T2 does ownership very differently and we don't support using different containers for checkpoints and ownership.

Reply:
It's a bad name, it's just the container name that we use. We'll adjust it when going through options cleanup.